Jenkinsfile

```diff
@@ -10,6 +10,14 @@
  * Contributors:
  *   See git history
  *******************************************************************************/
+def secrets = [
+  [path: 'cbi/org.eclipse.mylyn/develocity.eclipse.org', secretValues: [
+    [envVar: 'DEVELOCITY_ACCESS_KEY', vaultKey: 'develocity-token']
```

Comment:
can you change the value of vaultkey?

```
[envVar: 'DEVELOCITY_ACCESS_KEY', vaultKey: 'api-token']
```

Reply:
Of course, I only thought it should be `develocity-token` because of [this comment](https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/5409#note_2950414)

Reply:
I admit this is confusing. The `develocity-token` is a Jenkins cred, while your configuration retrieves the secret directly from our secretsmanager. What you did is the preferred way.